site/src/site/sphinx/en/tutorial-for-beginners.md

```diff
@@ -0,0 +1,282 @@
+## A Tutorial for Beginners
+
+### Prepare the demo
+
+Save the following demo to `demo/Demo.java` to have the first try on Arthas. 
+
+<sub>(If you are using Unix/Linux/Mac, then the commands below can be used.)</sub>
+
+```bash
+# you'd better find a proper location for the tutorial
+$ mkdir demo && touch demo/Demo.java
+
+# if there are indentation issues
+# ":set paste" in vim command mode to enter insert & paste mode
+$ vim demo/Demo.java 
+# ":set nopaste" afterwards to reset
+# ":wq" to save and exit
+```
+
+Copy and paste the following code to `demo/Demo.java`:
+
+```java
+package demo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+public class Demo {
+    static class Counter {
+        private static AtomicInteger count = new AtomicInteger(0);
+        public synchronized static void increment() {
+            count.incrementAndGet();
+        
+        }
+        public synchronized static int value() {
+            System.out.println("Processing...");
+            return count.get();
+        
+        }
+    
+    }
+
+    public static void main(String[] args) throws InterruptedException {
+        for (int i = 0; i < 5; ++i) {
+            new Thread(() -> {
+                    try {
+                    while (true) {
+                        Counter.increment();
+                        System.out.println(Thread.currentThread().getName() + " counter: " + Counter.value());
+                        testListAdd();
+                        TimeUnit.SECONDS.sleep(5);
+                    
+                    }
+                
+                    } catch (InterruptedException ignored) {
+                    ignored.printStackTrace();
+                
+                    }
+            
+                    }).start();
+        
+        }
+    
+    }
+
+    private synchronized static void testListAdd() {
+        System.out.println(Thread.currentThread().getName() + " running list add test");
+        List<String> list = new ArrayList<String>();
+        list.add("a");
+        list.add("b");
+
+        List<String> list2 = new ArrayList<String>();
+        list2.add("c");
+        list2.add("d");
+
+        int len = addTwoLists(list, list2);
+    
+    }
+
+    private synchronized static <T> int addTwoLists(List<T> list1, List<T> list2) {
+        list1.addAll(list2);
+        return list1.size();
```

Comment:
doesn't seem like these two methods need synchronized keyword, list1 and list2 both are local variable and don't need to be guarded